qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1298,14 +1298,16 @@ def _decode_job(self, raw_data: Dict) -> RuntimeJob:
         Returns:
             Decoded job data.
         """
-        hub = raw_data.get("hub")
-        group = raw_data.get("group")
-        project = raw_data.get("project")
-        instance = (
-            to_instance_format(hub, group, project)
-            if all([hub, group, project])
-            else None
-        )
+        instance = None
+        if self._channel == "ibm_quantum":
+            hub = raw_data.get("hub")
+            group = raw_data.get("group")
+            project = raw_data.get("project")
+            instance = (
+                to_instance_format(hub, group, project)
+                if all([hub, group, project])
+                else None
+            )
```

Comment:
```suggestion
            if all([hub, group, project]):
                instance = to_instance_format(hub, group, project)
```

Since `instance` is already `None` it might be cleaner to do it this way